src/ResourceManager/Resources/Commands.Resources.Test/Commands.Resources.Test.csproj

```diff
@@ -717,6 +717,9 @@
     <None Include="SessionRecords\Microsoft.Azure.Commands.Resources.Test.ScenarioTests.RoleAssignmentTests\RaGetByScope.json">
```

Comment:
@darshanhs90 Can you add a snippet to the changelog to ensure this is picked up for the release? Other than that the PR looks good.

Reply:
Updated the changelog